Pipfile

```diff
@@ -12,6 +12,7 @@ mopidy-moped = "*"
 mopidy-gmusic = "*"
 mopidy-pandora = "*"
 mopidy-youtube = "*"
+mopidy-local = "*"
```

Comment:
is this plus sign an artifact of GitHub meaning you added a line or is it in the file itself? Also, would you be able to add mopdiy-mpd?

Reply:
I think the error has something to do with the pipfile.lock being out of date. I have finally re-installed windows so have more space and a clean environment. when i try to build this with mopidy-local and mopidy-mpd in the pipfile i get the following error:
 [ 5/11] RUN set -ex  && pipenv install --system --deploy --python=$(which python3):
#9 0.391 + which python3
#9 0.393 + pipenv install --system --deploy --python=/usr/bin/python3
#9 0.830 Your Pipfile.lock (0cddf3) is out of date. Expected: (e3a56d).
#9 0.831 [DeployException]:   File "/usr/local/lib/python3.7/dist-packages/pipenv/vendor/click/decorators.py", line 73, in new_func
#9 0.831 [DeployException]:       return ctx.invoke(f, obj, *args, **kwargs)
#9 0.831 [DeployException]:   File "/usr/local/lib/python3.7/dist-packages/pipenv/vendor/click/core.py", line 610, in invoke
#9 0.831 [DeployException]:       return callback(*args, **kwargs)
#9 0.831 [DeployException]:   File "/usr/local/lib/python3.7/dist-packages/pipenv/vendor/click/decorators.py", line 21, in new_func
#9 0.831 [DeployException]:       return f(get_current_context(), *args, **kwargs)
#9 0.831 [DeployException]:   File "/usr/local/lib/python3.7/dist-packages/pipenv/cli/command.py", line 253, in install
#9 0.831 [DeployException]:       site_packages=state.site_packages
#9 0.831 [DeployException]:   File "/usr/local/lib/python3.7/dist-packages/pipenv/core.py", line 2063, in do_install
#9 0.831 [DeployException]:       keep_outdated=keep_outdated
#9 0.831 [DeployException]:   File "/usr/local/lib/python3.7/dist-packages/pipenv/core.py", line 1251, in do_init
#9 0.831 [DeployException]:       raise exceptions.DeployException
#9 0.831 ERROR:: Aborting deploy
------
executor failed running [/bin/sh -c set -ex  && pipenv install --system --deploy --python=$(which python3)]: exit code: 1

Reply:
In this PR the only thing you actually are suggesting to change/merge is this added line. Is that really your intension?

Reply:
I am not the author of the PR but the latest version of Mopidy seems to have dropped the mopidy-local and mopidy-mpd. this PR is adding mopidy-loacl back in but i would also like mopidy-mpd to be added.

Reply:
> I am not the author of the PR but the latest version of Mopidy seems to have dropped the mopidy-local and mopidy-mpd. this PR is adding mopidy-loacl back in but i would also like mopidy-mpd to be added.

Yes i'm trying to restore local and mpd, but this is my first PR so I'd probably missed something as @sasgoose commented , i also commented in other issue that i need to manually add local and mpd to make it work again in my docker setup.
Thanks.

Reply:
After this merge mopidy-local is working well... but I needed to add manually mopidy-mpd to connect HA:
python3 -m pip install Mopidy-mpd
Can we add this to the pipfile?

Reply:
I've created another pull request: https://github.com/wernight/docker-mopidy/pull/49